app/views/layouts/card_layout.html.haml

```diff
@@ -0,0 +1,29 @@
+
+!!! 5
+%html.no-js(lang="en")
+  %head
+    %meta(charset="utf-8")
+    %meta(http-equiv="X-UA-Compatible" content="IE=edge")
+    %meta(name="viewport" content="width=device-width, initial-scale=1")
+    %meta{:name => "robots", :content => "noindex"}
+    - if Settings.facebook_admins
```

Comment:
Does this need to be here? As you're using a separate layout for the card then there's no reason to have this facebook stuff here.